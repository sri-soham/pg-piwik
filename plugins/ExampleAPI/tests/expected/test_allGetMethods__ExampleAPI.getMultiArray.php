a:2:{s:10:"Limitation";a:2:{i:0;s:57:"Multi dimensional arrays is only supported by format=JSON";i:1;s:16:"Known limitation";}s:16:"Second Dimension";a:7:{i:0;b:1;i:1;b:0;i:2;i:1;i:3;i:0;i:4;i:152;i:5;s:4:"test";i:6;a:1:{i:42;s:3:"end";}}}