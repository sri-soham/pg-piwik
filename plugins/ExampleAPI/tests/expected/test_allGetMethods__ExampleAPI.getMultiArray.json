{"Limitation":["Multi dimensional arrays is only supported by format=JSON","Known limitation"],"Second Dimension":[true,false,1,0,152,"test",{"42":"end"}]}